src/sinks/new_relic_logs.rs

```diff
@@ -149,7 +148,6 @@ impl NewRelicLogsConfig {
             // The default throughput ceiling defaults are relatively
             // conservative so we crank them up for New Relic.
             concurrency: (self.request.concurrency).if_none(Concurrency::Fixed(100)),
-            rate_limit_num: Some(self.request.rate_limit_num.unwrap_or(100)),
```

Comment:
I think we'll want to update (that is, delete the override):
 
https://github.com/timberio/vector/blob/ea8f85602ab221a9a624497f0587b14fa8c47232/docs/cue/reference/components/sinks/new_relic_logs.cue#L39

And update the defaults:

https://github.com/timberio/vector/blob/ea8f85602ab221a9a624497f0587b14fa8c47232/docs/cue/reference/components.cue#L362-L371

I think this could be worth calling out as a release highlight since it does noticeably change the behavior. I think we could loop it in with a highlight for enabling ARC by default, but given this is going in first, would you mind adding a highlight that we could then update later? You can copy and update one of the ones in https://github.com/timberio/vector/tree/master/docs/content/en/highlights

Reply:
I don't properly understand cue yet but I have tried to make the adjustments as suggested.

Reply:
I think it would be updating that to `rate_limit_num:     uint64 | *100`

Reply:
@spencergilbert are you referring to `new_relic_logs.cue`? The default for `rate_limit_num` in that sink is now whatever the default in `TowerRequestConfig` is.